CIME/test_scheduler.py

```diff
@@ -681,15 +683,16 @@ def _create_newcase_phase(self, test):
                         )
                     )
                 elif case_opt.startswith("V"):
-                    self._cime_driver = case_opt[1:]
```

Comment:
The original code looks like a bug. The presence of one test with Vdriver would change the driver for all the tests.

Reply:
That's been there for a long time, seems like we would have seen problems by now if that were the case.   On the other hand, I agree.

Reply:
You mean adding one Vdriver a test in a suite would change all tests in that suite?

Reply:
@rljacob , yes. `self._cime_driver` is shared for all tests.